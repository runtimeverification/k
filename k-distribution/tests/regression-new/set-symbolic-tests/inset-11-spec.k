// Copyright (c) Runtime Verification, Inc. All Rights Reserved.
// generated by gen-tests.sh

requires "set-tests.k"

module INSET-11-SPEC
    imports SET-TESTS

    claim <k> inSet ( Y:MyId in ( SetItem(Z:MyId) SET:Set SetItem(X:MyId) ) ) => . </k>

endmodule
