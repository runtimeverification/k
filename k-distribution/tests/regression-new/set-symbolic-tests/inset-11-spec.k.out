  <k>
    inSetResult ( Y:MyId in SET
    SetItem ( X:MyId )
    SetItem ( Z:MyId ) ) ~> .K
  </k>
#And
  {
    false
  #Equals
    X:MyId in SET
    SetItem ( Z:MyId )
  }
#And
  {
    false
  #Equals
    Z:MyId in SET
  }
[Error] Prover: backend terminated because the configuration cannot be rewritten further. See output for more details.
